openedx/jobs/openedxReleaseJob.groovy

```diff
@@ -0,0 +1,42 @@
+package openedx
+
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+/* stdout logger */
+Map config = [:]
+Binding bindings = getBinding()
+config.putAll(bindings.getVariables())
+PrintStream out = config['out']
+
+// This is the job DSL responsible for creating the main pipeline job.
+pipelineJob('openedx-release-ci') {
+
+    definition {
+
+        parameters {
+            stringParam('OPENEDX_RELEASE_NAME', 'test-release',
+                        'open-release/${OPENEDX_RELEASE_NAME}')
```

Comment:
Most of this file I don't understand :)   Does this stringParam mean that the default will be "open-release/test-release"?

Reply:
Yep! That's what the default is. Happy to change if you have a preference. Also, this is where the Jenkinsfile in repo-tools gets the branch name for its command.